Code/Framework/AzToolsFramework/AzToolsFramework/Manipulators/RotationManipulators.cpp

```diff
@@ -158,4 +158,22 @@ namespace AzToolsFramework
     {
         m_circleBoundWidth = circleBoundWidth;
     }
+
+    AZStd::vector<BaseManipulator*> RotationManipulators::getManipulators()
```

Comment:
Per the Coding Standard the function name should be [upper camel case](https://github.com/o3de/sig-core/blob/main/governance/Coding-Standards-and-Style-Guide.md#function-names).

Also Side Note: I am working on change to add support for C++[23 range conversion constructors](https://www.open-std.org/jtc1/sc22/wg21/docs/papers/2022/p1206r7.pdf) and utility functions which would make this code a little bit simpler
i.e
```c++
auto UniquePtrToRawPtr = [](auto uniquePtr) { return uniquePtr.get(); };
AZStd::vector<BaseManipulator*> manipulators(AZStd::from_range, m_localAngularManipulators | AZStd::transform(UniquePtrToRawPtr));
manipulators.append_range(m_viewAngularManipulator | AZStd::transform(UniquePtrToRawPtr));
return manipulators;
```